src/rules/vouched-verification.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * @title Vouched Verification
+ * @overview Verify a person's identity using Vouched.
+ * @gallery true
+ * @category marketplace
+ *
+ * [Vouched](https://vouched.id) 
+ * 
+ * **ID Verification Process**
```

Comment:
TODO: Pull in the description from Marketplace and link to the installation guide.